main.py

```diff
@@ -884,6 +895,23 @@ def parse_args(self) -> None:
             required=False
         )
 
+        parser.add_argument(
+            "--oauthTokenUrl",
+            help="Url to authenticate that grants the access_token.",
+            type=str,
+            default=None,
+            required=False
+        )
+
+        parser.add_argument(
+            "--oauthParameters",
+            help="Parameters to be added to the oauth token request. (eg grant_type).",
```

Comment:
there are usually more than one - maybe show a fake example of what the exact parameter string would be

Reply:
e.g. "grant_type=authorization_code&scope=openid" (or whatever the form is supposed to be)

Reply:
sure, was thinking of adding to the readme some snippets on different authentications for this purpose on this ticket https://soos.atlassian.net/browse/PA-6608

Reply:
done